fix(heureka): fixes details panel deep link handling (#471)
cloudoperators/juno: .changeset/purple-nails-juggle.md

---
"@cloudoperators/juno-app-heureka": minor
"@cloudoperators/juno-app-greenhouse": patch
---

> This PR fixes the issue where reloading the app while the details panel is open would cause the panel to show a "busy" state indefinitely. The fix ensures that the details panel information is fully handled within the useUrlState hook, allowing for proper state restoration and display upon reload.

Reviewer: The description you write here is added to the changelog for this app. It shouldn't reference a PR because from the changelog you can't look at at it. Also it shouldn't be overly verbose. It makes sense to have a detailed description in the PR itself but not in the changelog.
```suggestion
Fixes issue with details panel being stuck in loading state when reloading the app while the panel is open
```
